clients/browser/index.html

```diff
@@ -404,7 +404,7 @@
 
         _balanceChanged(balance) {
             if (!balance) balance = Nimiq.Balance.INITIAL;
-            this._wltBalance.innerText = Nimiq.Policy.satoshisToCoins(balance.value).toFixed(Math.log10(Nimiq.Policy.SATOSHIS_PER_COIN));
+            this._wltBalance.innerHTML = '<span class="icon-nimiq"></span> ' + Nimiq.Policy.satoshisToCoins(balance.value).toFixed(Math.log10(Nimiq.Policy.SATOSHIS_PER_COIN));
```

Comment:
My problem with that was that the currency symbol then even displayed during initialization when no balance (not even 0.0000) is displayed. It looked odd. But that's my only reason :D